Neos.Neos/Tests/Functional/Fusion/NodeHelperTest.php

```diff
@@ -116,53 +124,55 @@ protected function buildView()
     protected function setUp(): void
     {
         parent::setUp();
+        $this->setUpCRRegistry();
+        $contentRepositoryId = ContentRepositoryId::fromString('default');
+        $this->iInitializeContentRepository($contentRepositoryId->value);
```

Comment:
```suggestion
        $contentRepositoryId = ContentRepositoryId::fromString('default');
```

imo we dont need the behat cr initialization stuff here?

Reply:
Well as the Helper needs a Registry, we should make use of the new capabilities. Perhaps an alternative less coupled to Behat would be nice...

Reply:
ah you mean the nodeHelper is used to ask the cr of the nodeTypeManager, to get the nodeType?

Reply:
exactly, no direct calls to Node::nodetype anymore